proto/spire/api/server/entry/v1/entry.proto

```diff
@@ -46,6 +46,21 @@ service Entry {
     // The caller must present an active agent X509-SVID. See the Agent
     // AttestAgent/RenewAgent RPCs.
     rpc GetAuthorizedEntries(GetAuthorizedEntriesRequest) returns (GetAuthorizedEntriesResponse);
+
+
+    // Syncs authorized entries down the caller. The caller controls which
```

Comment:
Syncs authorized entries down 'to' the caller.?

Reply:
Thanks :)